etc/tinyproxy.conf.in

```diff
@@ -205,6 +205,14 @@ Allow ::1
 # users.
 #BasicAuth user password
 
+# BasicAuthRealm : In case BasicAuth is configured .. the "realm" information.
```

Comment:
it seems the ".." syntax is superfluous in the config template. just use a comma.

Reply:
Same. Will update